spec/models/season_spec.rb

```diff
@@ -232,11 +232,27 @@
     end
   end
 
-  describe '.all_years' do
-    it 'return array of years' do
+  describe '.active_and_previous_years' do
+    it 'returns array of years that includes all active and previously active years' do
+      next_year = (Date.today.year + 1).to_s
       create :season, name: '2015'
       create :season, name: '2016'
-      expect(Season.all_years).to match_array ['2015', '2016']
+      create :season, name: next_year
+      expect(Season.active_and_previous_years).to match_array ['2015', '2016']
+    end
+
+    it 'returns current year only if current season is active season' do
+      before_acceptance_notification = Time.utc(2018, 5, 1, 23, 59, 58)
+      after_acceptance_notification = Time.utc(2018, 5, 2, 0, 0, 0)
+      create :season, name: '2015'
+      create :season, name: '2016'
+      create :season, name: '2018'
+
+      Timecop.freeze(before_acceptance_notification)
```

Comment:
Bonus points if the specs checks the right order of the years (see my previous comment).

Reply:
@klappradla We should definitely avoid that! I went with the block syntax because my understanding is that to place the the two Timecop functions in before hooks, we would need to break out the spec into two separate specs (so as to have control of when the time is set in relation to the two assertions). On the hook approach: would it suffice to simply add an `after` hook with the `Timecop.return`? I'm happy to revisit this if your preference is to avoid the block syntax.

@F3PiX I will revise the spec to test for exact array ordering.